lib/membrane_rtc_engine/endpoints/webrtc/rtp_connection_allocator.ex

```diff
@@ -146,6 +163,8 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPConnectionAllocator do
     # without question
     Process.monitor(pid)
 
+    Logger.debug("Track receiver registered")
```

Comment:
To make this log useful we'd need to add some id (I'd favor track id)

Reply:
It's actually a debugging leftover